Warning [-Wsign-compare]  in <CGAL/NewKernel_d/Vector/array.h>

```
/mnt/testsuite/include/CGAL/NewKernel_d/Vector/array.h:68:22: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
      CGAL_assertion(d==std::distance(f,e));
                      ^
```

--- a/NewKernel_d/include/CGAL/NewKernel_d/Vector/array.h
+++ b/NewKernel_d/include/CGAL/NewKernel_d/Vector/array.h
@@ -65,7 +65,7 @@ template<class NT_,class Dim_,class Max_dim_=Dim_> struct Array_vector {
 		struct Iterator {
 			template<typename Iter>
 				Vector operator()(unsigned CGAL_assertion_code(d),Iter const& f,Iter const& e) const {
-					CGAL_assertion(d==std::distance(f,e));
+					CGAL_assertion(d==(unsigned) std::distance(f,e));
 					CGAL_assertion(d<=d_);
 					//TODO: optimize for forward iterators
 					Vector a;

